java/pom.xml

```diff
@@ -10,6 +10,27 @@
     <packaging>pom</packaging>
 
     <name>Lance Parent</name>
+    <description>LanceDB Java API</description>
+    <url>http://lancedb.com/</url>
+
+    <developers>
+        <developer>
+            <name>Rong Rong</name>
+            <email>rong@apache.org</email>
+        </developer>
```

Comment:
please add a list of developers here. ideally this should be a group account IMO

Reply:
We have `dev@lancedb.com` group

https://github.com/lancedb/lance/blob/main/python/pyproject.toml#L5

Reply:
adjusting